data/etcd/store_test.go

```diff
@@ -15,9 +15,9 @@ const (
 )
 
 var (
-	config server.Config = server.Config{
+	config = server.Config{
 		Port:          ":50101",
-		EtcdEndpoints: []string{"http://localhost:2379"},
+		EtcdEndpoints: []string{"http://etcd:2379"},
```

Comment:
As much as I want CI tests to pass, I hate to break local tests which we use during development, and I don't want it to be error prone changing this and then remember to change it back. I suppose a quick and dirty way to fix for local development right now might be to add an entry to the HOSTS file that maps `etcd` to 127.0.0.1. I don't see any other easy ways besides running the tests themselves from a container attached to the swarm, but that involves a somewhat tedious build step. Thoughts?